vulcand/registry.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	localEtcdProxy = "http://127.0.0.1:2379"
+	LocalEtcdProxy = "http://127.0.0.1:2379"
```

Comment:
I would say if it is duplication in tests then that is ok. It does not seem right to export something just to make it accessible in tests.